### PR TITLE
Editable table jahz

### PR DESCRIPTION
**Pull request by jahzielv**
Created a new DataTable component that loads all the data from a cost center, then updates the DB when changes are made to it.

### CODE REVIEW

File: src/components/DataModificationPage.js [hunk 1]
```diff
@@ -17,192 +17,203 @@ import TableRow from '@material-ui/core/TableRow';
...
+//             {button}
+
+//           </Grid>
+
+//         </Grid>
+//       </React.Fragment>
+//     );
+//   }
+// }
+
```
Comment (mhelmstadter) [inline src/components/DataModificationPage.js]: Are we going to need the old code anymore?

File: src/components/DataTable.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        // if (added) {
+        //     const startingAddedId = rows.length > 0 ? rows[rows.length - 1].id + 1 : 0;
+        //     rows = [
+        //         ...rows,
+        //         ...added.map((row, index) => ({
+        //             id: startingAddedId + index,
+        //             ...row,
+        //         })),
+        //     ];
+        // }
```
Comment (mhelmstadter) [inline src/components/DataTable.js]: can these comments be deleted?

File: src/components/DataTable.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        if (changed) {
+            rows = rows.map(row => (changed[row.id] ? { ...row, ...changed[row.id] } : row));
+            // rows.map(row => (this.setState({ changedRows: this.state.changedRows.push(row.id) })));
+            // console.log(this.state);
```
Comment (mhelmstadter) [inline src/components/DataTable.js]: how about these?

File: src/components/DataTable.js [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+            var i;
+            for (i = 1; i <= rows.length; i++) {
+                if (changed[i] != null) {
+                    // console.log(changed[i]);
```
Comment (mhelmstadter) [inline src/components/DataTable.js]: commented out

File: src/components/DataTable.js [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+                    var copy = this.state.changedRows;
+                    copy.push(i);
+                    this.setState({ changedRows: copy });
+                }
+            }
+        }
+        // if (deleted) {
```
Comment (mhelmstadter) [inline src/components/DataTable.js]: commented out

**Review by mhelmstadter — DISMISSED** [review]
Make sure you merge develop into this branch now that develop has changed. Other than that it looks good, I'll test on my machine after it's all up to date!

File: src/components/DataModificationPage.js (hunk 1 above)
Comment (jahzielv) [reply src/components/DataModificationPage.js]: Nope, should be good to delete it.

File: src/components/DataTable.js (hunk 2 above)
Comment (jahzielv) [reply src/components/DataTable.js]: Yeet

File: src/components/DataTable.js (hunk 3 above)
Comment (jahzielv) [reply src/components/DataTable.js]: also yeet

**Review by emanuallan — APPROVED** [review]

**Pull request merged**